src/dotnet/Fable.Compiler/Replacements.fs

```diff
@@ -2537,8 +2536,9 @@ let checkLiteral com fileName range (value: obj) (typ: Fable.Type) =
     match typ with
     | Fable.Enum("System.Text.RegularExpressions.RegexOptions") ->
         match value with
-        | :? int as i when i = int RegexOptions.IgnoreCase
-                        || i = int RegexOptions.Multiline
-                        || i = int RegexOptions.ECMAScript -> ()
+        | :? int as i when i = 0x0001 // "i" - RegexOptions.IgnoreCase
+                        || i = 0x0002 // "m" - RegexOptions.Multiline
+                        || i = 0x0100 // "e" - RegexOptions.ECMAScript
```

Comment:
This is not necessary. The IL code is exactly the same.

Reply:
@inosik The point is to not use anything from the regex namespace (in this case, RegexOptions) to lower the API surface.